frontend/ui/elements/screensaver_menu.lua

```diff
@@ -43,7 +42,6 @@ return {
                 enabled_func = function()
                     return G_reader_settings:readSetting("screensaver_type") == "cover"
                             or G_reader_settings:readSetting("screensaver_type") == "random_image"
-                            or G_reader_settings:readSetting("screensaver_type") == "image_file"
```

Comment:
You are removing all references to `G_reader_settings:readSetting("screensaver_type")` possible being `"image_file"`.
What will happen to people who have it set as `"image_file"`?
Does it need a migration?
And I would rather keep `image_file` (which I think more people may use) than `document_cover`.